docs/getting-started/playground/start.md

```diff
@@ -0,0 +1,107 @@
+---
+id: start
+---
+
+# Gno Playground
+
+## Overview
+
+The Gno Playground is a web editor and sandbox that enables developers to 
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
+- **A keypair in a Gno.land wallet, such as [Adena](https://adena.app)**
+
+To fully utilize Playground features, you will need to install a Gno.land web
+browser wallet, such as [Adena](https://www.adena.app/), and create a keypair. This will allow you to
+interact with the Playground.
+
+## Playground Features
+
+To get started, visit the Playground at [play.gno.land](https://play.gno.land). You will be greeted with a
+simple `package.gno` file:
+
+![default_playground](../../assets/getting-started/playground/default_playground.png)
+
+The Playground has the following features:
+- `Share` - Get a permanent, shareable link to the code currently found in the playground
+- `Deploy` - Connect your wallet and deploy your code to a Gno.land network
+- `Format` - Format your Gno code
+- `Run` - Run a specific expression on your code
+- `Test` - Run tests on your code
+- `REPL` - Experiment and troubleshoot in real-time using the GnoVM with
+interactive REPL features (experimental)
+
+Let's dive into each of the Playground features.
+
+### Share
+
+This feature allows users to get a permanent shortlink to the code found in the
+playground at the time of clicking. This way, Gno code can be shared easily. 
+
+### Deploy
+
+Deploy allows users to quickly deploy their Gno code to the chain. After connecting 
+a Gno.land wallet, users will be able to choose which package path to deploy to,
+as well as which network.
+
+![default_deploy](../../assets/getting-started/playground/default_deploy.png)
+
+After inputting your desired package path, you can select the network you would 
+like to deploy to, such as [Portal Loop](../../concepts/portal-loop.md) or local,
+and click deploy.
+
+:::info
+Even if you don't have testnet tokens, the Playground will automatically provide
+you with enough to cover the gas cost at the time of deployment.
+:::
+
+### Format
+Format will format your code, using `gofmt`.
```

Comment:
Should we provide more context? Something like 
```
The 'Format' feature utilizes the Monaco editor and [`gofmt`](https://pkg.go.dev/cmd/gofmt) to automatically refine and standardize your Gno code's syntax.
```

Reply:
I was hesitant to include mentioning "Monaco" since it doesn't seem to me that a lot of people know about it, but in this case more info is good imo. [b2cb449](https://github.com/gnolang/gno/pull/1773/commits/b2cb4499caf700d588306ac892a5d3cddbb1abd0)